src/pages/Landing.jsx

```diff
@@ -35,9 +37,15 @@ function Landing() {
 		if (day !== setupTime) {
 			localStorage.removeItem("Focus");
 			localStorage.removeItem("Todos");
+			localStorage.removeItem("Quote");
 			localStorage.setItem("setupTime", day);
 		}
 	}
+	const navigate = useNavigate();
+	const changeNameHandler = () => {
+		localStorage.removeItem("name");
+		navigate("/");
+	};
 	useEffect(() => {
 		const user = localStorage.getItem("name");
 		dispatch({ type: "SET_USERNAME", payload: { value: user } });
```

Comment:
suggestion: might create an object in constants for storing all the actions and passing the variable here